src/micromamba/devcontainer-feature.json

```diff
@@ -9,12 +9,26 @@
             "type": "string",
             "proposals": [
                 "latest",
-                "1"
+                "1.0.0"
```

Comment:
I think it is better not to have this, given that it will soon become outdated.
Details on how to specify the version can be included in the `NOTE.md` file.
e.g. https://github.com/rocker-org/devcontainer-features/tree/main/src/pandoc

Reply:
Well 1.0.0 is just a generic example since the current version is 1.1.0. Is it okay, or should we delete `proposals` entirely?

Reply:
I think it would be better to delete `"1.0.0"`, leaving only the `"latest"`.

Reply:
Ok, we can delete it.

It's hard for me to understand what should go here because I don't understand what the `proposals` are used for. The specification says:

> A list of suggested string values.

But who or what is doing the suggesting? Is there an autocomplete somewhere? Or is it a form of in-place documentation which is functionally disregarded?

Reply:
As you understand it, I don't think this is currently handled by any clients (VSCode or Codespaces).
The users cannot know unless they look at the json file directly.

Reply:
Thanks, that makes sense.

I don't have any objection, but I don't understand the logic... wouldn't it be nice to have a semver example? Also, if we had soft version matching, would you leave `"1"` in, even though that would eventually become outdated?

Reply:
I believe that specifying major versions is worthwhile to avoid breaking changes.
On the other hand, I do not see much value in specifying up to minor versions, as they quickly become outdated.

Since it is difficult for users to know if a certain Feature actually has soft version matching functionality, I think it is best to include an example in NOTES.md. (although it would be worthwhile if major versions or LTS could be specified as proposals may be processed by clients in the future)
https://github.com/rocker-org/devcontainer-features/blob/1befccc67e978e248c4c5c87808fd8bdfb157d62/src/quarto-cli/NOTES.md?plain=1#L29-L48

Reply:
Specifying the full version number is very valuable to ensure reproducibility. Although in theory it shouldn't happen, often patch releases will break something. Then the environment which worked yesterday doesn't work today, and nobody knows why.

Reply:
I removed `"1.0.0"` from `proposals` and added an explanation in `NOTES.md`.

It still seems strange to me that we remove `"1.0.0"` but I don't think it matters, so if you're satisfied, let's resolve this conversation.